R/create_protocol.R

```diff
@@ -0,0 +1,233 @@
+#' @title Create a folder with a markdown template to start working on a new fieldwork protocol
+#'
+#' @description This function will create a new folder based on values that are passed on via the parameters and creates a R-markdown skeleton based on a template file to start working on a new protocol.
+#'
+#' @param title A character string giving the main title of the protocol
+#' @param subtitle A character string for an optional subtitle
+#' @param short_title A character string of less than 20 characters to use in folder and filenames
+#' @param authors A character string for authors of the form Voornaam Achternaam and multiple authors separated by a comma
+#' @param date A character string of the date in ISO 8601 format (YYYY-MM-DD)
+#' @param reviewers A character string for reviewers of the form Voornaam Achternaam and multiple authors separated by a comma
+#' @param file_manager A character string for the name of the document maintainer
+#' @param revision A semantic version number of the form major.minor.patch. For development versions a fourth component is appended starting from .9000. The default is 0.0.0.9000 and should normally not be changed.
+#' @param projectname A character string giving the project name. Best to use an acronym for this.
+#' @param theme A character string equal to generiek (default), water, lucht, bodem, vegetatie of soorten.
+#'
+#' @return A new subfolder beneath src with an Rmarkdown file
+#'
+#' @importFrom rprojroot find_root is_git_root
+#' @importFrom stringr str_replace_all str_extract
+#' @importFrom assertthat assert_that is.string is.date
+#' @importFrom rmarkdown draft render
+#' @importFrom purrr map_chr
+#' @importFrom whisker whisker.render
+#'
+#' @export
+#'
+#' @examples
+#' \dontrun{
+#' create_sfp()
+#'}
+create_sfp <- function(
+  title = "Titel",
```

Comment:
```suggestion
  title,
```
No need to specify defaults as you are forcing the user to enter some value

Reply:
Nice, didn't know that.